cli/tests/unit/permissions_test.ts

```diff
@@ -13,12 +13,25 @@ Deno.test(async function permissionInvalidName() {
   }, TypeError);
 });
 
+Deno.test(function permissionInvalidNameSync() {
```

Comment:
Separate tests have been created for `querySync()`. Should the test functions just be appened to the existing async tests or are they good as they are?

Reply:
This seems fine